src/igl/vulkan/RenderPipelineState.cpp

```diff
@@ -392,28 +392,30 @@ VkPipeline RenderPipelineState::getVkPipeline(
   IGL_PROFILER_FUNCTION_COLOR(IGL_PROFILER_COLOR_CREATE);
 
   // @fb-only
-  const VkDescriptorSetLayout DSLs[] = {
```

Comment:
I have an issue here importing this PR. Please keep the `DSLs` array definition outside of the `if`-block so that the `// @fb-only` marker above is unmodified. Once you do it, I can try reimporting this PR.